test/playwright/utils/breakpoints.ts

```diff
@@ -133,13 +133,15 @@ const describeEachBreakpoint =
   }
 
 const describeEvery = describeEachBreakpoint(Array.from(SCREEN_SIZES.keys()))
+const describeDesktopMobile = describeEachBreakpoint(['lg', 'sm'])
```

Comment:
This name is kind of confusing :sweat_smile: I'd have a hard time understanding the difference between this an `describeEvery` without looking at the implementation.

Is this a pattern you think we'd use in lots of different e2e tests, only testing `lg` and `sm` specifically? If not, the tests themselves can call `breakpoints.describeEachBreakpoint`.

What's the goal behind testing only `lg` and `sm` instead of using `describeEvery`?

Reply:
The goal was to save some space :) And yes, I couldn't find a better name.
I removed it in the end, and used `describeEvery` instead.